### ISSUE TITLE
Make kPa better w/ counting fractions & clean up formulas

### ISSUE THREAD

**Issue by jaredhasenklein** [jaredhasenklein/2017-FRC-Resources-Issue-Tracking-Only-#7]
[Reported from scca229 via CD](https://www.chiefdelphi.com/forums/showpost.php?p=1627288&postcount=17):

> The counting of the fuel for Match points need to be corrected to be additive between the all types (Auto High/Low, TeleOp High/Low) and not separate scoring lines.
> 
> Examples directly from 4.3 Scoring in the Manual (beginning with TeleOp to remove Auto issues and using 1/9ths for clarity):
> 
> 12 in Low, 5 in High = 3 kPa, 3 Match Points
> 12/9 + 5/3 = 3
> 1 3/9 + 1 6/9 = 3
> 
> 14 in low, 5 in High = 3 2/9 kPa, 3 Match Points
> 14/9 + 5/3 = 3 2/9
> 1 5/9 + 1 6/9 = 3 2/9
> 
> The calculator is showing both examples as 2 Match Points because it is treating each goal as a separate scoring line when the manual says that they are added together via examples. You don't just lop off the remainder in each goal before adding them together. At most, you would be off by 1 Match Point in the TeleOp part of the calculator, but see 2016 Einstein for how important a single point can be. The additive error doesn't end there though and can add another point of error when Auto is included.
> 
> The example given in the Scoring rule is 17 Low in Auto gives 5 2/3 kPa. An additional 3 Low in TeleOp will add 3/9 (1/3) to that giving 6 kPa. The calculator is not adding them together that way resulting in only the 5 still (it takes adding 7 to TeleOp low to make it add another point).
> 
> The kPa column should be showing the value in 1/9ths for TeleOp, 1/3rds for Auto to correctly show what the running total is without rounding. The Match Points column should probably not include the fuel as without having all 4 pieces of information together and calculated the total can be off by up to 2 points if the remainders add up to over 2 additional kPa:
> 
> Auto Low - 2 fuel - 2/3 kPa
> TeleOp High - 2 fuel - 2/3 kPa
> TeleOp Low - 8 fuel - 8/9 kPa
> Total is 2 2/9 kPa or 2 Match Points. When put in the calculator this way, it shows 1 Match point and that is on the Low TeleOp line for some reason.
> 
> 
> 
> TLDR; The Match Points for the goals should not be separate lines but total for all with their respective weights based on goal and when scored. Calculate:
> (Auto Low/3) + (Auto High) + (TeleOp Low/9) + (TeleOp High/3) for kPa (1RP if >40) and then round down to whole number for Match Points from fuel.

**Comment by jaredhasenklein** [comment]
Replaced 
`f12` with `=D12*1`
`f13` with `=D13/3`
`f16` with `=D16/3`
`f17` with `=(D17/9)`

Also replaced Total formula with `=CONCATENATE(SUM(E11:E19), " Match pts, ", TEXT(F20,"# #/#")," kPa, & ",FLOOR(SUM(IF(F20>40,1,0),IF(D18=4,1,0)),3), " RP(s)*")`

Now, fractions are shown.
